common/components/componentsBySection/VolunteerActivityReporting/VARFormTitle/stories/VARFormTitle.stories.jsx

```diff
@@ -0,0 +1,14 @@
+//@flow
+
+import React from 'react';
+import VARFormTitle from '../VARFormTitle';
+
+export default {
+    title: 'VolunteerActivityReporting/VARFormTitle',
+    component: VARFormTitle,
+};
+
+
+export const Default = (): React.Node => {
```

Comment:
Having to pass the rendering function in a story is an old pattern.  Please do it like this:
```
export const Default = {args: {}};
```